logrotate.c

```diff
@@ -2195,6 +2195,7 @@ static int writeState(const char *stateFilename)
 	if (bytes < 0)
 		error = bytes;
 
+/* time in seconds it takes earth to go around sun */
```

Comment:
I see, the comment and commit message should have been more verbose that is now fixed in f6a06b3d53b10007f05e524aa2f2449340241c4e. BTW it would be nice to hear from Jan Kaluza why he chose to use exact solar year here.